db/version_set.h

```diff
@@ -426,6 +427,7 @@ class VersionStorageInfo {
                               // is guaranteed to be empty.
   // Per-level max bytes
   std::vector<uint64_t> level_max_bytes_;
+  std::vector<uint64_t> level_max_bytes_static_;
```

Comment:
I don't see the reason for this field, it seems they are always kept in sync.

Reply:
`max_bytes_for_level_base` and `max_bytes_for_level_multiplier` are changeable dynamically, so it should be calculated every round.